### ISSUE TITLE
[Eclipse] Copyright Header

### ISSUE THREAD

**Issue by DominikPinsel** [eclipse-dataspaceconnector/DataSpaceConnector#18]
We need to include a copyright header in every source file, see
https://www.eclipse.org/projects/handbook/#ip-copyright-headers

```
/********************************************************************************
 * Copyright (c) {year} {owner}[ and others] 
 *
 * This program and the accompanying materials are made available under the 
 * terms of the Apache License, Version 2.0 which is available at
 * https://www.apache.org/licenses/LICENSE-2.0
 *
 * SPDX-License-Identifier: Apache-2.0
 * 
 * Contributors: 4
 *   {name} - {description}
 ********************************************************************************/
```

<sub> Dominik Pinsel <dominik.pinsel@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>

**Comment by beardyinc** [comment]
I can do this in a one-shot commit with Intellij.
- Is the initial copyright holder `Microsoft Corporation`?
- Should the `Contributors` section be initially left blank or should it be:
    `Contributors: 1`
    `Microsoft Corporation - initial commit`

that would result in the following header:

```
/*
 * Copyright (c) 2021 Microsoft Corporation
 *  
 *  This program and the accompanying materials are made available under the 
 *  terms of the Apache License, Version 2.0 which is available at
 *  https://www.apache.org/licenses/LICENSE-2.0
 *  
 *  SPDX-License-Identifier: Apache-2.0
 *  
 *  Contributors: 1
 *       Microsoft Corporation - initial commit
 *
 */
```

**Comment by MoritzKeppler** [comment]
and don't miss to adjust the LICENSE file :-)

**Comment by jimmarino** [comment]
Some small modifications (year and contributor description):

/*
 * Copyright (c) 2020, 2021 Microsoft Corporation [and others]
 *  
 *  This program and the accompanying materials are made available under the 
 *  terms of the Apache License, Version 2.0 which is available at
 *  https://www.apache.org/licenses/LICENSE-2.0
 *  
 *  SPDX-License-Identifier: Apache-2.0
 *  
 *  Contributors: 
 *       Microsoft Corporation - initial API and implementation
 *
 */

**Comment by jimmarino** [comment]
Also, [and others] is not added at first but as soon as it is modified, it is appended

**Comment by MoritzKeppler** [comment]
solved by #29
